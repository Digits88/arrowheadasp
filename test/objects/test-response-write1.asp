This tests output using Response.Write

<% Response.Write("Hello Mom!!!") %><br>
