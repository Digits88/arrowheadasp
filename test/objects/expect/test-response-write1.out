This tests output using Response.Write

Hello Mom!!!<br>
